### ISSUE TITLE
CI: org.elasticsearch.backwards.Backwards50ClientYamlTestSuiteIT test {p0=indices.clear_cache/10_basic/clear_cache with request_cache set to false}

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#23702]
Two failures on master:
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+java9-periodic/1879
https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+periodic/2181

For some reason I had trouble opening the full log but the error mails contain the following information:

java.lang.AssertionError: Failure at [indices.clear_cache/10_basic:12]: expected [2xx] status code but api [indices.clear_cache] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"illegal_argument_exception","reason":"request [/_cache/clear] contains unrecognized parameter: [request]","stack_trace":"[request [/_cache/clear] contains unrecognized parameter: [request]]; nested: IllegalArgumentException[request [/_cache/clear] contains unrecognized parameter: [request]];\n\tat org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:617)\n\tat org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:562)\n\tat org.elasticsearch.rest.BytesRestResponse.build(BytesRestResponse.java:144)\n\tat org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:101)\n\tat org.elasticsearch.rest.BytesRestResponse.<init>(BytesRestResponse.java:92)\n\tat org.elasticsearch.rest.RestController.dispatchRequest(RestController.java:203)\n\tat 

```
java.lang.AssertionError: Failure at [indices.clear_cache/10_basic:23]: did not get expected warning header [
	Deprecated field [request_cache] used, expected [request] instead
]

	at __randomizedtesting.SeedInfo.seed([B8561E3323C7D293:300221E98D3BBF6B]:0)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:377)
	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:357)
	at jdk.internal.reflect.GeneratedMethodAccessor15.invoke(Unknown Source)
```

@nik9000 could this still be related to https://github.com/elastic/elasticsearch/commit/798c19dd7f3a1f45db4f44f687b2684309c26490 ?

**Comment by nik9000** [comment]
I've kicked off a snapshot build for 5.x. That should fix it.

**Comment by nik9000** [comment]
Lots of similar failures are waiting for the CI build.... Sorry!

**Comment by nik9000** [comment]
The snapshot is failing so I've pushed 490d29f4fc15d8d156d814a49194e619b687b96a which should get us back to stable.

**Comment by nik9000** [comment]
These ought to go away. I'm going to leave this open until I revert 490d29f and we get our tests back.

**Comment by nik9000** [comment]
Closed by b3dca364ced3b67c5fdb835ce9b4b5fdf5125a0e.
